code/index.html

```diff
@@ -3,28 +3,131 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
     <title>Project 3</title>
     <!-- dont forget to add a css file and link it here! -->
   </head>
   <body>
+    
     <!-- video or image as a header is cool :) -->
+    <header class="hero-image">
+      
+      <div class="inner-hero-header"> 
+        <nav class="navbar">
+          
+          <a href="#" class="logo">Frusto</a>
+          <ul class="nav-menu">
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">About us</a>
+            </li>
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">News</a>
+            </li>
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">Blog</a>
+            </li>
+            
+            <li class="nav-item">
+              <a href="#" class="nav-link">Contact</a>
+            </li>
+          
+          </ul>
+          </nav>
+      </div>
+
+      
+      <div class="logo-place">
+        <a href="#" class="logo-mobil">Frusto</a>
+      </div>
```

Comment:
I was wondering why you had your logo twice - once as `.logo` and once as `.logo-mobil`. I saw that you used `display:none` on the `.navbar`, which hid your logo as well. One potential solution in the future would be to use `display:none` on your `.nav-item` class instead, which would hide the links, so you don't have to insert the logo twice in your HTML and CSS. Another option is to take place the logo before the `<nav>` class, and then hide the entire `.navbar`. That way, you don't have to be worried about the logo being duplicated twice in the code, and any styling can be solved using @media queries.